pandas/io/excel/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pandas.io.excel._base import read_excel, ExcelWriter, ExcelFile
+from pandas.io.excel._openpyxl import _OpenpyxlWriter
+from pandas.io.excel._util import register_writer
+from pandas.io.excel._xlsxwriter import _XlsxWriter
+from pandas.io.excel._xlwt import _XlwtWriter
+
+__all__ = ["read_excel", "ExcelWriter", "ExcelFile"]
+
+
+register_writer(_OpenpyxlWriter)
```

Comment:
Could obviously make this more succinct with a loop but again attempting to minimize diff

Reply:
I'm not sure I follow - given that +/- 2000 line diff from `git`, what diff are you trying to minimize?